dms_version/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017-2020 MuK IT GmbH
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Document Management System Version",
+    "summary": """Version Control for Documents""",
+    "version": "13.0.1.0.0",
+    "category": "Document Management",
+    "license": "LGPL-3",
+    "website": "http://github.com/OCA/dms",
+    "author": "MuK IT,Odoo Community Association (OCA)",
+    "depends": ["dms"],
+    "data": [
+        "security/ir.model.access.csv",
+        "views/file.xml",
+        "views/version.xml",
+        "views/storage.xml",
+    ],
+    "application": False,
```

Comment:
```suggestion
    "application": False,
    "external_dependencies": {"python": ["bsdiff4"]},
```
Need to add bsdiff4 dependency to addon work fine.

Reply:
The bsdiff4 is not a strict requirement. You can use the version control without it as well. Only if you want to compress your versions the dependency ist needed.

Reply:
If it is not required, maybe we can do two modules (just to avoid this kind of checks), `dms_version` and `dms_version_incremental`, that will implement the incremental part. What do you think?

Reply:
I personally think that directly the library requirement should be mandatory and don't maintain 2 versions of code, as if you need versioning, the diff/incremental way is always more efficient, and the requirement is very light (only a library). WDYT?

Reply:
IMO: To make an extra module seems excessive for such a small extension. I have no problem making it a strict requirement. Technically it is not, but I guess most will use incremental versions to reduce the needed disk space.

Reply:
Yeah, that's why I say directly to not have other option except incremental.

Reply:
One version then :wink: